src/store/UserStore.js

```diff
@@ -34,36 +32,123 @@ class store {
 
 		mobileActive: false,
 		slackActive: false
-		           
-    }; 
 
-	setState(updates) {
+	};
+	notify = () => {
+		for (const handler of this.handlers) {
+			handler(this.state)
+		}
+	}
+
+	setState = (updates) => {
 		Object.assign(this.state, updates)
 		this.notify()
 	}
 
-	//------
 	// Subscriptions
 
 	handlers = new Set()
 
-	subscribe(handler) {
+	subscribe = (handler) => {
 		this.handlers.add(handler)
 		handler(this.state)
-
 		return {
 			remove: () => {
 				this.handlers.delete(handler)
 			}
 		}
 	}
 
-	notify() {
-		for (const handler of this.handlers) {
-			handler(this.state)
+
+	loadUsers = () => {
+		// console.log(this.state.users[1])
+		fetch('http://localhost:3000/api/users', {
```

Comment:
This url is `flaky` => https://www.vocabulary.com/dictionary/flaky

Works locally not in the internet 🕸